irods/manager/__init__.py

```diff
@@ -13,3 +13,6 @@ def server_version(self):
 
     def __init__(self, sess):
         self.sess = sess
```

Comment:
Maybe this can call `_set_manager_session`?

Reply:
seems harmless, will examine closer.